### PR TITLE
use JSON array to hold deployer information

### PR DESCRIPTION
**Pull request by vigneshmsft**
change `Deployers` format to `[{"displayName":, "slackUserId": }]` to support mentions in slack notifications.

### CODE REVIEW

File: spec/lib/deployers_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Deployers do
   around do |ex|
-    ClimateControl.modify DEPLOYERS: 'one,two,three,four,five,six,seven' do
+    ClimateControl.modify DEPLOYERS: '[{"name":"One","userId":"1"},{"name":"Two","userId":"2"},{"name":"Three","userId":"3"},{"name":"Four","userId":"4"},{"name":"Five","userId":"5"}]' do
```
Comment (duncanjbrown) [inline spec/lib/deployers_spec.rb]: What are `name` and `userId`? From Slack? How do we get them?

Comment (vigneshmsft) [reply spec/lib/deployers_spec.rb]: name: is just a display name,
userId: is the slack userId of a team member, which I have configured in the the Azure DevOps library group
You can view the slack userId of a person by navigating to their profile from inside slack.

Comment (duncanjbrown) [reply spec/lib/deployers_spec.rb]: Ah ok. Thanks. Could we rename them something more explicit like `nameOnDashboard` and `slackUserId`?

Comment (vigneshmsft) [reply spec/lib/deployers_spec.rb]: @duncanjbrown renamed

File: lib/state.rb [hunk 2]
```diff
@@ -52,6 +52,14 @@ def deployers_for_today
     Deployers.for_today
   end
 
+  def todays_deployer
+    Deployers.for_today[0]['displayName']
+  end
+
+  def todays_reserves
```
Comment (duncanjbrown) [inline lib/state.rb:59]: :+1: this is better!

**Review by duncanjbrown — APPROVED** [review]

**Pull request merged**